app/posts/cross-programme/simplifying-ect-training-options.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Simplifying training options for early career teachers 
+description: "Why the training options for ECTs have been simplified and how it's changed the digital services "
+date: 2025-04-23
+---
+
+## How the training options are changing
+
+Historically, there’ve been 3 training options for schools to choose from to train their early career teachers (ECTs): 
+1. A training programme led by lead providers and delivered by delivery partners, with an online learning platform and training events. 
+
+2. Using learning materials created by lead providers but otherwise running their own training programme as a school. 
+
+3. Creating their own learning materials and running their own training programme as a school. 
```

Comment:
I _think_ it'll be fine but usually I'd leave a blank line between paragraphs and lists and no lines between list items. From memory the markdown renderer we use on the design history wraps list items that have spaces between then in additional `<p>`  elements that gives them extra spacing.

Reply:
I think i've fixed this :)